File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/user/SaveKanrenshaPersonCapsuleDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.user;

import java.io.Serializable;

import net.seijishikin.jp.normalize.common_tool.dto.FrameworkCapsuleDto;
import net.seijishikin.jp.normalize.manage.kanrensha.dto.kanrensha.KanrenshaPersonDto;

/**
 * 関連者個人格納Dto
 */
public class SaveKanrenshaPersonCapsuleDto extends FrameworkCapsuleDto implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 関連者個人Dto */
    private KanrenshaPersonDto kanrenshaPersonDto = new KanrenshaPersonDto();

    /**
     * 関連者個人Dto
     *
     * @return 関連者個人Dto
     */
    public KanrenshaPersonDto getKanrenshaPersonDto() {
        return kanrenshaPersonDto;
    }

    /**
     * 関連者個人Dto
     *
     * @param kanrenshaPersonDto 関連者個人Dto
     */
    public void setKanrenshaPersonDto(final KanrenshaPersonDto kanrenshaPersonDto) {
        this.kanrenshaPersonDto = kanrenshaPersonDto;
    }

}

File: app/front/src/components/main/dto/kanrensha/saveKanrenshaPersonCapsuleDto.ts
import { FrameworkCapsuleDto, type FrameworkCapsuleDtoInterface } from "seijishikin-jp-normalize_common-tool";
import { KanrenshaPersonDto, type KanrenshaPersonDtoInterface } from "./kanrenshaPersonDto";

interface SaveKanrenshaPersonCapsuleDtoInterface extends FrameworkCapsuleDtoInterface {
    /** 関連者個人Dto */
    kanrenshaPersonDto: KanrenshaPersonDtoInterface;
}


class SaveKanrenshaPersonCapsuleDto extends FrameworkCapsuleDto
    implements SaveKanrenshaPersonCapsuleDtoInterface {

    /** 関連者個人Dto */
    kanrenshaPersonDto: KanrenshaPersonDtoInterface;

    constructor() {
        super()

        this.kanrenshaPersonDto = new KanrenshaPersonDto();
    }
}

export { type SaveKanrenshaPersonCapsuleDtoInterface, SaveKanrenshaPersonCapsuleDto }
